rollup.config.js

```diff
@@ -10,26 +10,42 @@ const packageJson = require('./package.json');
 const peerDependencies = Object.keys(packageJson.peerDependencies);
 const dependencies = Object.keys(packageJson.dependencies);
 
+function globals() {
```

Comment:
I think we can remove `globals` from `baseUmdConfig` now. It's not used anymore

Reply:
I got this warning when running `npm run build`:


```
(!) You have passed an unrecognized option
Unknown input option: globals. Allowed options: acorn, acornInjectPlugins, cache, chunkGroupingSize, context, experimentalCacheExpiry, experimentalOptimizeChunks, experimentalTopLevelAwait, external, inlineDynamicImports, input, manualChunks, moduleContext, onwarn, perf, plugins, preserveModules, preserveSymlinks, shimMissingExports, treeshake, watch
```

Removing `globals` from the config will also fix this.